testdata/dbc/example/example.dbc

```diff
@@ -39,6 +39,10 @@ BO_ 500 IODebug: 6 IO
  SG_ TestBoolEnum : 32|1@1+ (1,0) [0|0] "" DBG
  SG_ TestScaledEnum : 40|2@1+ (2,0) [0|6] "" DBG
 
+BO_ 600 IOFloat32: 8 IO
+	SG_ Float32ValueNoRange : 0|32@1- (1,0) [0|0] "" DBG
+	SG_ Float32WithRange : 32|32@1- (1,0) [-100|100] "" DBG
```

Comment:
Please follow the same kind of indentation as the rest of the DBC file (i.e. with one space indentation and not one tab). The difference is visible in editors with different tab stops :)

```suggestion
 SG_ Float32ValueNoRange : 0|32@1- (1,0) [0|0] "" DBG
 SG_ Float32WithRange : 32|32@1- (1,0) [-100|100] "" DBG
```

Reply:
Otherwise great test additions! :raised_hands:

Reply:
Can you, please, point me to example of the negative test in the repo? I will add some similar tests.

Reply:
I'm afraid there aren't any yet, but it's basically what `internal/generate/compile_test.go`does, but with invalid DBC snippets

Reply:
Thanks, I will take a look.

Reply:
Added a couple negative tests.

Reply:
Great, thank you!